scripts/core/editor3/components/toolbar/StyleButton.jsx

```diff
@@ -35,7 +35,7 @@ const StyleTooltips = {
     'ordered list': gettext('Ordered list'),
     TH: gettext('Toggle Header'),
     suggestions: gettext('Toggle Suggestions Mode'),
-    invisibles: gettext('Toggle Non-Printing characters')
+    invisibles: gettext('Toggle formatting marks')
```

Comment:
here it's using wrong key after the change above

Reply:
It's still called `invisibles` in the code, the change is only for the UI. I've checked that hovering the icon displays the new title.

Reply:
I see, my bad